adal/src/main/java/com/microsoft/aad/adal/PackageHelper.java

```diff
@@ -59,6 +59,23 @@ public PackageHelper(Context ctx) {
         mAcctManager = AccountManager.get(mContext);
     }
 
+    static boolean isCompanyPortalInstalled(final PackageManager pkgManager) {
+        return isPackageInstalled(pkgManager, "com.microsoft.windowsintune.companyportal");
```

Comment:
>com.microsoft.windowsintune.companyportal [](start = 47, length = 41)

should this be in a config file?

Reply:
The pkg names of published apps won't change after store-submission so we can safely ship these values as constants. I'll convert these fields to `private static final` for cleanliness. Thanks 👍

Reply:
Resolved